.firebaserc

```diff
@@ -1,6 +1,6 @@
 {
   "projects": {
-    "default": "explore-gemini"
+    "default": "explore-gemini-dev"
```

Comment:
Unrelated change, attempt to erase the unused deployment in firebase.

Reply:
Yeah, that won't work. It's the name of the overarching project, not the site. We can't really remove the default site. I'll revert.